src/js/charts/PieChart.js

```diff
@@ -119,7 +122,11 @@ export default class PieChart extends AggregationChart {
 			let title = (this.formatted_labels && this.formatted_labels.length > 0
 				? this.formatted_labels[i] : this.state.labels[i]) + ': ';
 			let percent = (this.state.sliceTotals[i] * 100 / this.state.grandTotal).toFixed(1);
-			this.tip.setValues(x, y, {name: title, value: percent + "%"});
+			let valueTooltip = percent + "%";
+			if(this.config.formatTooltip){
```

Comment:
Any reason to not reuse the formatTooltipX and formatTooltipY?

Reply:
I didn't want to confuse this single type of values of aggregate charts with the charts legend X and Y cause in aggregate the concept of X and Y is not present.

Reply:
`formatTooltip` itself is very generic, can we name it better? `formatAggrTooltip` is something that I can think of now, do you have any suggestions?

Reply:
Beyond this, the feature works like a breeze, we're good to merge it.